docs/UI/AddSubMenu/index.md

```diff
@@ -42,7 +42,8 @@ UI.AddSubMenu represents a request from an application to add a sub-menu to the
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
 |menuID|Integer|true|minvalue: 1<br>maxvalue: 2000000000|
-|menuParams|[Common.MenuParams](../../common/structs/#menuparams)|true||
+|menuParams|Common.MenuParams|true||
```

Comment:
Is there a reason the links is broken here? Is it possible for us to include a link to the struct?

Reply:
@mrapitis, thank you.  
https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/102/commits/2b598604d41af289d2fa6366226d976f4335fd90  added relative links to params, so that links work correctly on https://smartdevicelink.com after merging PR